src/stream/src/executor/exchange/input.rs

```diff
@@ -74,9 +74,8 @@ impl LocalInput {
     }
 
     #[cfg(test)]
-    pub fn for_test(channel: Receiver<Message>) -> BoxedInput {
-        // `actor_id` is currently only used by configuration change, use a dummy value.
-        Self::new(channel, 0).boxed_input()
+    pub fn for_test(actor_id: ActorId, channel: Receiver<Message>) -> BoxedInput {
+        Self::new(channel, actor_id).boxed_input()
     }
```

Comment:
There seems no difference between this function and `new`. 😄 How about removing it?

Reply:
> There seems no difference between this function and `new`. 😄 How about removing it?

resolved